TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -286,7 +283,12 @@ class RouteOptionsViewController: UIViewController, UITableViewDelegate, UITable
     // MARK: Process data
 
     func searchForRoutes() {
-        if let startingDestination = searchFrom as? CoordinateAcceptor, let endingDestination = searchTo as? CoordinateAcceptor{
+        // If no date is set then date should be same as today's date
+        if routeSelection.datepickerButton.titleLabel?.text == "Leave now" {
```

Comment:
Is there a more stable way to check this?

Reply:
unfoturnately not